compiler/vm/vmrunner.nim

```diff
@@ -225,6 +230,23 @@ proc registerCallbacks(c: var TCtx): bool =
       echo "expected '$#' callback but got '$#'" % [p.string, other[i].string]
       result = false
 
+proc createLegacyStackTrace(
+    c: TCtx, 
+    thread: VmThread, 
+    instLoc: InstantiationInfo = instLoc(-1)
+  ): VMReport =
+  # TODO: duplicated from `compilerbridge`, consolidate after all the reports
+  #       bits are untangled.
```

Comment:
The stack-trace is directly reported in the case of `vmrunner` -- maybe it makes sense to instead duplicate the formatting logic to here and not use `localReport`? This is only a suggestion, it's fine if it stays like it is for now.